examples/safe_remote_purchase/safe_remote_purchase.v.py

```diff
@@ -0,0 +1,38 @@
+#Safe Remote Purchase (https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst) ported to viper and optimized
+
+#Rundown of the transaction:
+#1. Seller posts item for sale and posts safety deposit of double the item value. Balance is 2*value.
+#(1.1. Seller can reclaim deposit and close the sale as long as nothing was purchased.)
+#2. Buyer purchases item (value) plus posts an additional safety deposit (Item value). Balance is 4*value
+#3. Seller ships item
+#4. Buyer confirms receiving the item. Buyer's deposit (value) is returned. Seller's deposit (2*value) + items value is returned. Balance is 0.
+
+value: public(wei_value) #Value of the item
```

Comment:
@DavidKnott actually, since `wei_value` is the lowest divisible unit, perhaps it IS a good idea to ensure it is a multiple of 2 e.g. `assert (msg.value % 2) == 0` in `__init__()`. What do you think?

Reply:
You're right, even though Viper supports decimals the `wei_value` requires the `assert (msg.value % 2) == 0`.
My bad @chgue.